autonormalize/tests/test_example.py

```diff
@@ -7,6 +7,8 @@ def test_ft_mock_customer():
     df = ft.demo.load_mock_customer(n_customers=80, n_products=50, n_sessions=200,
                                     n_transactions=10000, return_single_table=True)
 
+    df.ww.init()
+
     entityset = an.auto_entityset(df, name="Customer Transactions", time_index='transaction_time')
 
     assert set(entityset['transaction_id'].columns) == set(['transaction_id', 'session_id', 'transaction_time',
```

Comment:
cannot find test case for function normalize_entity in autonormalize.py

Reply:
yes, here are my concerns:
1, As this will fix https://github.com/alteryx/autonormalize/issues/39 which is mentioned in PR description, and in issue #39 had got AttributeError when run normalized_es = an.normalize_entity(es), so to add a test case for function normalize_entity will be much helpful. 
2, And even through you passed all tests, it because those test cases were not including function normalize_entity. No mater what reason the test case of function normalize_entity was missing, it's a good time to add it now

Reply:
got it, good point